### PR TITLE
fix: add aparna2071

### LINKED ISSUE #1249: Aparna Soneja
## Profile Screenshot

Screenshot 2022-07-21 222832

### PR DESCRIPTION
**Pull request by aparna2071**
resolves #1249 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/aparna2071.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/54634804/180271682-7083563f-cb70-4b00-b562-158849d7bea5.png",
+  "issueId": 1249 ,
```
Comment (Panquesito7) [inline profiles/aparna2071.json:3]: There's an extra space in here.
```suggestion
  "issueId": 1249,
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/aparna2071.json (hunk 1 above)
Comment (aparna2071) [reply profiles/aparna2071.json:3]: Thanks @Panquesito7

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Review by kabir0x23 — APPROVED** [review]
Looks good ✅

**Comment by aparna2071** [comment]
Can you please merge my PR ?

**Comment by aparna2071** [comment]
Thanks @Panquesito7

**Pull request merged**